interface/super/show_groups_popup.php

```diff
@@ -48,8 +48,10 @@
 <h1><?php xl('Layout groups','e'); ?></h1>
 <ul>
 <?php
-$res = sqlStatement("SELECT distinct(group_name) FROM layout_options WHERE " .
-                    "form_id = '".$_GET['layout_id']."' ORDER BY group_name");
+
+$query .= "SELECT distinct(group_name) FROM layout_options WHERE form_id = ? ORDER BY group_name";
```

Comment:
If you want to go multi-line (which is good for line length and reviews of the query elements) you can always just end your line of the query by putting in a space, closing the quote, adding a dot then adding the next quoted line vs. declaring the variable again with a .=
That space is important.  I personally like breaking out FROM, WHERE, AND, statements as much as practical unless it is a short line with AND ((statement) OR (statement)).
  I also like the concat dot and space at the beginning of the line, such as
"FROM layout_options"
." WHERE (statement);";
It is best to end your query strings with a semi-colon within the quotes.  I there is optional content, I prefer to concat a .";"; after all criteria is added, just so that no matter what, the final closing semicolon for the query and the final PHP statement end are always treated properly.